controllers/expression.js

```diff
@@ -0,0 +1,48 @@
+import {
+  insertExpression,
+  fetchExpressions,
+  removeExpression,
+} from '../service/expression';
+
+export const addExpression = async (request, response) => {
+  try {
+    const { value } = request.body;
+    const { userId } = request.user;
+
+    await insertExpression(userId, value);
+
+    return response.status(200).json({
```

Comment:
Shouln't this be a `201 Created` since you are adding a new expression?

Reply:
And it is a good practice to return the created/updated entity if it was succesfully created.